### PR TITLE
[Refactor] ADF-363 Avoid relying on html structure for conditional logic in resourcemngr

### PR DESCRIPTION
**Pull request by 404passionFruit**
Jira
https://oat-sa.atlassian.net/browse/ADF-363

Related PRs
https://github.com/oat-sa/tao-core/pull/2968

Env
Link in the ticket

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
         "decimal.js": "10.1.1",
         "devbridge-autocomplete": "^1.2.20",
         "dompurify": "^1.0.10",
+        "eve": "git+https://github.com/adobe-webplatform/eve.git#eef80ed",
```
Comment (404passionFruit) [inline package.json:57]: This is added to fix CI tests issues on install stage

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by clagosarias — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-svetlana.playground.kitchen.it.taocloud.org:43581/ - https://oat-sa.atlassian.net/browse/ADF-352 is working as expected

**Pull request merged**